build/common.targets

```
@@ -54,6 +54,21 @@
       <Import Project="$(NuGetBuildTasksPackTargets)" />
   </ImportGroup>
 
+  <!-- Projects we pack become public APIs that others can use -->
+  <ItemGroup Condition=" '$(PackProject)' == 'true' and '$(NoPublicAPIs)' != 'true' ">
+    <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" PrivateAssets="all" />
+
```

Can you add a condition to avoid calling the API analyzer in source-build mode?

```suggestion
  <ItemGroup Condition='$(IsBuildOnlyXPLATProjects)' != 'true'">
    <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" PrivateAssets="all" />
  </ItemGroup>    
  <ItemGroup Condition=" '$(PackProject)' == 'true' and '$(NoPublicAPIs)' != 'true' ">
```